episodes/chapter4.md

```diff
@@ -194,6 +132,9 @@ You will see a message:
 >
 > Please distribute each of these keys privately to each of the organizations. Note that they may reset their API key so that no-one but them knows it.
 
+You will need this API key to register the nodes of the organizations in the collaboration. For more details, see [the chapter of setting up a vantage6 node](./chapter6.md).
```

Comment:
I would not say `register` - in my view that is what you have just done by creating the collaboration and the node. I would rather say something like `You will need the API keys when you run the nodes to authenticate with the server`

Reply:
And I would say instead of the second sentence `We will discuss this tomorrow in [link to chapter 6]`

Reply:
Usually participants get the lesson material after the workshop. So it's better not to mention `tomorrow` ;-)

Reply:
In that case, maybe 'We will discuss this later in chapter 6' ;-)